cep-sigstore-predicate.md

```diff
@@ -0,0 +1,366 @@
+     claimant identity. The certificate issued by Fulcio is, in turn auditable
+     via [RFC 6962] Certificate Transparency (CT) logs.
+
+    - Alice signs her attestation with her ephemeral key, and distributes a
+      "bundle" containing both her attestation and her signing certificate.
+
+    - Instead of establishing trust a long-lived key from Alice, Bob establishes
+      trust in Alice's identity.
+
+    - Bob can verify the attestation's signature against Alice's emphemeral key,
+      which in turn can be verified as authentically Alice's via the Fulcio-
+      issued certificate.
+
+    With this flow, neither Alice nor Bob needs to maintain long-lived signing
+    or verifying keyrings, in turn reducing the attacker surface for key
+    compromise.
+
+  Another key misuse-resistance contribution within Sigstore is *machine
+  identities*. A machine identity behaves similarly to a human identity
+  (Alice or Bob), but identifies a machine instead of a human. For example,
+  `github.com/example/example/.github/workflows/release.yml@refs/tags/v1.2.3`
+  could be the machine identity of a GitHub Actions workflow that ran from
+  `release.yml` within `example/example` against the `v1.2.3` tag.
+
+## Motivation
+
+The conda ecosystem contains metadata that answers the following questions,
+in part or in full:
+
+* _Who_ (or _what_) published this package?
+* _What_ is the package's hash?
+* _Where_ was this package _published from_, and where _to_?
+* _When_ was this package published?
+
+However, this metadata is not currently **cryptographically verifiable**:
+the consuming party must either trust it as presented, or verify it manually
+against independent sources of truth (such as a project's release history).
+
+Attestations that present this metadata in a cryptographically
+verifiable manner are desirable for a number of reasons:
+
+* Package maintainers wish to demonstrate the integrity and authenticity
+  of their package uploads;
+* Individual downstream users wish to verify the integrity and authenticity of
+  packages they consume, without placing additional trust in the
+  channel or channel's hosting server;
+* Attestations change the sophistication and risk profile for attackers in
+  defenders' favor: the attacker must be sufficiently sophisticated
+  to access private key material, *and* have a risk tolerance profile that
+  accepts exposure via auditable transparency logs.
+
+More broadly, attestation schemes like the one proposed in this CEP have
+seen adoption in similar and related ecosystems:
+
+* Python (PyPA/PyPI): [PEP 740] and [PyPI - Attestations]
+* NodeJS (npm): [npm - Generating provenance statements]
+* Ruby (RubyGems): [rubygems/release-gem]
+
+## Specification
+
+### Attestation format
+
+This CEP proposes the following attestation statement layout, using the
+[in-toto Statement schema]:
+
+- `predicateType` **MUST** be `https://schemas.conda.org/attestations/publish/v1`
```

Comment:
Sorry if this is an obvious question, but what are the contents of https://schemas.conda.org/attestations/publish/v1? Or is it just a string identifying _this_ attestation format? i.e. is it empty?

Reply:
Let's go for the JSON schema as Jannis suggested, and let's open the relevant PR in https://github.com/conda/schemas to provide the file.